### DISCUSSION TITLE
Black Temple: Less trash, more bosses

### DISCUSSION THREAD

**Discussion by Capoguerra** [chromiecraft/chromiecraft#7274] [Suggestions]
Spending too much time on trash takes away time for bosses obviously and this can compromise the morale and success of a raid. BT will be the longest of the raids ever released and complicated to fix, do we want to try to make it as accessible as possible?

In this case, common sense should take priority over the blizzlike philosophy.

Proposal: could we leave the current hp of trash (without tuning) or decrease the total number of trash by 50% while maintaining tuning?

**Comment by BeefSteaker** [comment]
Thanks for the feedback. It's unlikely any mobs will be removed, but we'll see definitely consider keeping tuning light on trash. 

Keep an eye out for an announcement that tuning is up for testing, the more testing that gets done the better we can tune it to people's liking

**Comment by vvranceanu** [comment]
:)
